src/disco_spot/bot.py

```diff
@@ -61,6 +61,8 @@ def __init__(self):
                 scope=self.scope,
             )
         )
+        most_epic_playlist = self.sp.playlist(self.config.PLAYLIST_ID)
+        self.playlist_url = most_epic_playlist['external_urls']['spotify']
```

Comment:
It's a good argument whether or not to store the playlist_url on class instantiation -- I agree that we'd have to check if url is static (surely ...) . 

But in any case. I think it's more standard API design to just grab it on request and ONLY store the `PLAYLIST_ID` in our env file. 

I could be convinced otherwise, though. How much do we care about this bot?